emperor/support_files/js/controller.js

```diff
@@ -302,6 +331,61 @@ define([
 
   /**
    *
+   * Write save file for the current controller settings
+   *
+   **/
+   EmperorController.prototype.saveConfig = function() {
+    var saveinfo = {};
+    // Assuming single sceneview for now
+    sceneview = this.sceneViews[0];
+    saveinfo.cameraPos = sceneview.camera.position;
+    saveinfo.cameraQuat = sceneview.camera.quaternion;
+    //Save settings for each controller in the view
+     _.each(this.controllers, function(controller, index) {
+      if (controller === undefined) {
+        return;
```

Comment:
For simplicity, should we change this to be if controller !== undefined? Same in the loadConfig method.